fast_plotter/__main__.py

```diff
@@ -88,7 +88,7 @@ def process_one_file(infile, args):
             df_filtered.rename({weight: "sumw"}, axis="columns", inplace=True)
             df_filtered["sumw2"] = df_filtered.sumw
         else:
-            df_filtered = df.filter(like=weight, axis="columns").copy()
+            df_filtered = df.copy()
```

Comment:
Do you remember the reason to make this change?  I think you essentially remove the `weight` option here which allows you to only plot one of the event weight schemes if the dataframe contains several (e.g. when using the Extended Weights in the CHIP workflow).

Reply:
Not really. I can change the line back and see what happens. I think there was some logic behind this choice but I can't remember what it was

Reply:
I can't see how it could break the rest of the code if you have this in, so if you're ok to revert it let's do it.  If it needs changing back we can easily add this change back in a future PR.

Reply:
Ok, I removed that change and tried to re-run. It errors because I need the `n` column of the dataframe for the error calculation

Reply:
That makes sense.  Ok let's break the `weights` option for now and open an issue to solve this later on.  I'll merge this in now, thanks for the PR!